tiago_trajectory_controller/src/run_traj_control.cpp

```diff
@@ -103,7 +103,7 @@ void waypoints_arm_goal(control_msgs::FollowJointTrajectoryGoal& goal)
   goal.trajectory.points[index].velocities.resize(7);
   for (int j = 0; j < 7; ++j)
   {
-    goal.trajectory.points[index].velocities[j] = 0.25;
+    goal.trajectory.points[index].velocities[j] = 0.0;
```

Comment:
I have done some tests and setting velocity 0 at intermediate waypoints causes the robot to execute the trajectory in a step-wise fashion. By setting for example here velocity 1.0 causes the robot to execute the trajectory in a smoother way, i.e. not stopping in this waypoint. I think we can set here 1.0 and explain in the tutorial the effect of this and invite the reader to set this value to 0 and observe how the robot stops momentarily in the waypoint.